Bio/SeqRecord.py

```diff
@@ -250,23 +274,10 @@ def __init__(
         self.features = features
 
     # TODO - Just make this a read only property?
```

Comment:
This comment is obsolete with your proposed changes (but right now I don't see why you changed the SeqRecord, I though you were only changing the Seq classes).

Reply:
I will delete the comment. 
I mentioned in #3309 I wanted to try to implement `__slots__` for both classes. If this is a problem I can remove the changes for SeqRecord.